Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -486,7 +486,7 @@ BindablePropertyContext GetContext(BindableProperty property)
 			for (var i = 0; i < properties.Count; i++)
 			{
 				BindablePropertyContext context = properties[i];
-				if (ReferenceEquals(context.Property, property))
+				if (context != null && ReferenceEquals(context.Property, property))
```

Comment:
Good question. Can context is by null? 

If context deleted in `_propeties` then exception is be `ArgumentOutOfRangeException` is not `NullReferenceException`.

`BindablePropertyContext` is added in `_properties`. only in `CreateAndAddContext`. Maybe problem is in it?